### PR TITLE
[FIX] web: wysiwyg bug fixed

### PR DESCRIPTION
**Pull request by mgh-odoo**
bug fixed when quick save form that have html field.

Task:https://www.odoo.com/web#id=1932687&action=333&active_id=131&model=project.task&view_type=form&menu_id=4720
Pad:https://pad.odoo.com/p/r.0f1a04ce8913bdce7fccae973c067226

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/mass_mailing/static/src/js/mass_mailing_widget.js [hunk 1]
```diff
@@ -40,6 +40,9 @@ var MassMailingFieldHtml = FieldHtml.extend({
      */
     commitChanges: function () {
         var self = this;
+        if (!this.wysiwyg || this.loadedDef.state() !== 'resolved') {
```
Comment (Gorash) [inline addons/mass_mailing/static/src/js/mass_mailing_widget.js]: Use a flag instead a deferred and put it at true in _onLoadWysiwyg method.

**Comment by Gorash** [comment]
Please rebase and adapt this pr. Some fixes has been merged.
Thanks.

**Comment by mgh-odoo** [comment]
hello @Gorash 
branch rebased.

**Comment by Gorash** [comment]
Can you add a test please?
Thanks

**Comment by Gorash** [comment]
@mgh-odoo this fix is for saas-12.2 not for master

**Comment by mgh-odoo** [comment]
hello @Gorash 
mentioned testcase is getting failed everytime on runbot but working perfect locally through browser. i am not sure why is this happening. can you please have a look?
or is there any other way to prove that save method not going to call if user quickly edit and save(because not yet rendered)
https://github.com/odoo-dev/odoo/commit/22063cd42765fd8dc66ea77ff87115d589e99f20#diff-5d0e84f077e903f78c81eb9ccca9ff73R521
cc @dja-odoo

**Comment by Gorash** [comment]
I took your commit and adapted the test. It is enough, he checks that there is no error.
https://github.com/odoo/odoo/pull/31581
thanks

**Comment by Gorash** [comment]
#31581

**Pull request closed without merging**